### PR TITLE
Initial config

### PR DESCRIPTION
**Pull request by Asiderr**

### CODE REVIEW

File: MANUAL.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+3. Install the git package.
+
+```
+nix-env -iA nixos.git
+```
+
+4. Clone the 3mdeb nixpkgs fork.
+
+```
+git clone https://github.com/3mdeb/nixpkgs.git -b landing-zone
```
Comment (macpijan) [inline MANUAL.md:26]: @Asiderr There is no such branch anymore. Do you mean this MR: https://github.com/3mdeb/nixpkgs/pull/1 ? I can merge it  and let's use master here. using branch will quickly become obsolete.

File: MANUAL.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+7. Update the system
+
+Change the nixpkgs directory to the 3mdeb nixpkgs repository.
+
+```
+sudo nixos-rebuild switch -I nixpkgs=~/nixpkgs
+```
+
+8. Reboot the system
```
Comment (macpijan) [inline MANUAL.md:49]: Let's include command for reboot as well

File: MANUAL.md (hunk 1 above)
Comment (pietrushnic) [reply MANUAL.md:26]: @macpijan @Asiderr LZ should have tags and release process, tags should be not removed from tree ever.

Comment (macpijan) [reply MANUAL.md:26]: I'm not sure if it has and it's not only LZ here, but multiple of packages. But yes, using tag for the blog post content would be optimal for reproducibility in the future.

**Comment by macpijan** [comment]
Moved to: https://github.com/3mdeb/nixos-trenchboot-configs/pull/2

**Pull request closed without merging**